src/stores/ui.ts

```diff
@@ -24,32 +29,42 @@ export interface UiState {
    * whether the site layout is desktop (or mobile).
    */
   isDesktopLayout: boolean
+  /**
+   * the screen's max-width breakpoint.
+   */
+  breakpoint: Breakpoint
   /**
    * whether the request user agent is mobile or not.
    */
   isMobileUa: boolean
 }
 
-const cookieOptions: CookieSerializeOptions = {
-  path: '/',
-  sameSite: 'strict',
-  maxAge: 60 * 60 * 24 * 60, // 60 days
-  secure: isProd,
-}
+export const screenSizes = [...Object.values(SCREEN_SIZES), 0]
+export const breakpoints = [...Object.keys(SCREEN_SIZES), 'xs']
```

Comment:
A comment might be nice to explain the 'fake' minimum values added to the lists here.